### ISSUE TITLE
change date display in data.js

### ISSUE THREAD

**Issue by kusj** [codrops/Calendario#44]
Hello,

Can the format in which the date in being displayed in data.js be changed ?

Ex to :
 '24-08-2015' : 'Event 1',
'24-08-2015' : 'Event 2',

Thanks,

**Comment by deviprsd21** [comment]
Yes it can be, in the new v5.0.1, visit https://github.com/deviprsd21/Calendario
